docs/coding/clients/README.md

```diff
@@ -0,0 +1,9 @@
+# Clients
+
+TigerBeetle has official client libraries for the following languages:
+
+- [.Net](/src/clients/dotnet/) ([nuget package](https://www.nuget.org/packages/tigerbeetle)).
+- [Go](/src/clients/go/) ([package](https://github.com/tigerbeetle/tigerbeetle-go), [API docs](https://pkg.go.dev/github.com/tigerbeetle/tigerbeetle-go)).
+- [Java](/src/clients/java/) ([maven central package](https://central.sonatype.com/artifact/com.tigerbeetle/tigerbeetle-java), [API docs](https://javadoc.io/doc/com.tigerbeetle/tigerbeetle-java/)).
+- [Node](/src/clients/node/) ([npm package](https://www.npmjs.com/package/tigerbeetle-node)).
+- [Python](/src/clients/python/) ([PyPi package](https://pypi.org/project/tigerbeetle/)).
```

Comment:
Add a note at the end that bug reports for any of these should go to https://github.com/tigerbeetle/tigerbeetle/issues

And also add back:

```
Subscribe to [this tracking issue](https://github.com/tigerbeetle/tigerbeetle/issues/2231) to be
notified when there are breaking API/behavior changes that are visible to the client.
```
somewhere.